### PR TITLE
KOGITO-3664: Add cypress test

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Review by vpellegrino — APPROVED** [review]
Good from my side. Thanks @jomarko

File: packages/online-editor/it-tests/support/commands.ts [hunk 1]
```diff
@@ -47,12 +47,15 @@ Cypress.Commands.add("loadEditor", () => {
   });
 });
 
-Cypress.Commands.add("ouiaId", { prevSubject: "optional" }, (subject, id: string, options = {}) => {
-  const idSelector = `[data-ouia-component-id='${id}']`;
+Cypress.Commands.add("ouia", { prevSubject: "optional" }, (subject, locator, options = {}) => {
```
Comment (jstastny-cz) [inline packages/online-editor/it-tests/support/commands.ts:50]: Just checking cypress commands here.

As long as IDE-based hinting works for the selector attribute I would consider this a superior solution to what we have elsewhere:
In kogito-apps we have 2 commands: ouiaId, ouiaType, each one for different mode of operation. So this raises a question if generally `ouia({ouiaId:'xy'})` is better than `ouiaId('xy')`, or more complex `ouia({ouiaId:'xy', ouiaType:'ab'})` compared to `ouiaId('xy', 'ab')`. Which I think it is (cause in the other solution the order of arguments is not clear).
For reference: https://github.com/kiegroup/kogito-apps/blob/main/ui-packages/packages/cypress-ouia/src/index.ts

But generally I like the approach here - when we're merging those cypress-ouia tools globally, your approach would be the one to adopt.

File: packages/online-editor/it-tests/integration/DMNExpressionEditor.ts [hunk 2]
```diff
@@ -23,49 +23,49 @@ describe("DMN Expression Editor Test", () => {
 
   it("Test New Expresssion editor - context", () => {
     // click Create new decision model button (new DMN)
-    cy.ouiaId("try-dmn-sample-button").click();
+    cy.ouia({ ouiaId: "try-dmn-sample-button" }).click();
```
Comment (jstastny-cz) [inline packages/online-editor/it-tests/integration/DMNExpressionEditor.ts:26]: I think the semantics define ouaiId as a distinction of several components of the same ouiaType. In this case we're not certain what the ouiaType is, so this is prone to error. ouiaId does not need to be globally unique, it's just supposed to be unique in surrounding context - the actual component. So let's say we have a table, with multiple rows. I would consider it a valid state when each of these rows contain a button with ouiaId='buttonSubmit' - it is uniquely distinguished component within surrounding context (the table row).
But this does not look like a problem inflicted by these changes.
More in https://ouia.readthedocs.io/en/latest/

File: packages/online-editor/it-tests/support/commands.ts (hunk 1 above)
Comment (jstastny-cz) [reply packages/online-editor/it-tests/support/commands.ts:50]: Thanks @jomarko for checking the IDE hinting support, it seems fine, so no problem with this command.

**Pull request merged**